runtime/protocol/aws-protocol-core/common/src/aws/smithy/kotlin/runtime/awsprotocol/ClockSkewInterceptor.kt

```diff
@@ -89,7 +89,8 @@ public class ClockSkewInterceptor : HttpInterceptor {
             try {
                 Instant.fromRfc5322(it)
             } catch (e: ParseException) {
-                logger.warn(e) { "Service returned malformed \"Date\" header value \"$it\", skipping skew calculation" }
+                logger.warn { "Service returned malformed \"Date\" header value \"$it\", skipping skew calculation" }
+                logger.debug(e) { e.message.toString() }
```

Comment:
**Nit**: Logging `e.message` is unnecessary since the logger should already include it. I'd just give a bit more context about where the error was encountered and leave it at that:

```kotlin
logger.debug(e) { "Error while parsing \"Date\" header from service response" }
logger.warn { "Service returned malformed \"Date\" header value \"$it\", skipping skew calculation" }
```